openedx/adg/lms/webinars/models.py

```diff
@@ -120,6 +120,8 @@ class Webinar(TimeStampedModel):
         blank=True,
         related_name='webinar_modified_by',
     )
+    zoom_id = models.CharField(verbose_name=_('Webinar Zoom Id'), max_length=100, blank=True, null=True, )
+    send_cancellation_emails = models.BooleanField(default=False, verbose_name=_('Send Cancellation Emails'), )
```

Comment:
What is the reason for adding this field?

Reply:
@asadali145 this field was added in the ticket because of the following screens:

[After cancelling event (in Zoom)](https://www.figma.com/file/z2RVPsiHp7FiB5N9WOVfVb/ADG-Phase-2?node-id=302%3A3019)
[After cancelling event (in Zoom)](https://www.figma.com/file/z2RVPsiHp7FiB5N9WOVfVb/ADG-Phase-2?node-id=302%3A3040) and after cancellation emails have been sent.

So according to these designs, we had to store whether cancellation email were sent or not against a cancelled webinar. However, after discussion with Abdulrahman and Anubhav in yesterday's standup, the cancellation emails would be sent immediately, at the time the webinar is cancelled and the "Send cancellation emails?" option would be removed from the dashboard of the cancelled webinars.

Therefore, now we no longer need this field.

Reply:
Thanks for the explanation. I kept that conversation in mind that is why I was confused.